src/network/netdev/netdev-gperf.gperf

```diff
@@ -140,7 +140,8 @@ Bond.PrimaryReselectPolicy,        config_parse_bond_primary_reselect,   0,
 Bond.ResendIGMP,                   config_parse_unsigned,                0,                             offsetof(Bond, resend_igmp)
 Bond.PacketsPerSlave,              config_parse_unsigned,                0,                             offsetof(Bond, packets_per_slave)
 Bond.GratuitousARP,                config_parse_unsigned,                0,                             offsetof(Bond, num_grat_arp)
-Bond.AllSlavesActive,              config_parse_unsigned,                0,                             offsetof(Bond, all_slaves_active)
+Bond.AllSlavesActive,              config_parse_bool,                    0,                             offsetof(Bond, all_slaves_active)
```

Comment:
hmm, this looks like an indepdendent bugfix. could you split this into a separate commit please (in the same PR, that'd be fine)